### DISCUSSION TITLE
Ruffle Help

### DISCUSSION THREAD

**Discussion by nwt21** [ruffle-rs/ruffle#14029] [Q&A]
Hello

I am new to downloading Ruffle and I was using the website WayBackMachine to watch aN anime music video on the website Vidiac.com (no longer online).

I was able to find the authors video channel, but when I clicked on a video I wanted to watch it didn't load. I've downloaded Firefox, Google Chrome and Safari while enabling the Ruffle Extension for each browner, but no luck.

Here are 3 videos I wanted to watch while trying to use the Ruffle Extension, but I have not had any luck. Any help would be greatly appreciated, or to be pointed in the right direction.

https://web.archive.org/web/20070422021351/http://www.vidiac.com/video/afde1a37-2647-4a5b-b3c6-3bf37decbffd.htm

https://web.archive.org/web/20070508220015/http://www.vidiac.com/video/D89AAC71-0B62-4EC3-946F-60586CD220BE.htm

https://web.archive.org/web/20070430022321/http://www.vidiac.com/video/379F50EF-5184-48A4-8A38-E517176F4DBD.htm

**Comment by Croworbit** [comment]
there may be a possibility that these videos were not archived fully by the wayback machine, and are now lost

**Reply by nwt21** [reply]
That would be a bummer. 

if anyone who reads this is able to watch these videos and let me know if it works, that would be amazing. I'm only trying to find a song that was used on one of these videos, but I've tried using Shazam, and Google AI to try and "hum" the song, but no luck.

**Comment by wtotino** [comment]


**Reply by nwt21** [reply]
Thank you for taking the time to check this out. I respect and appreciate all of your help. 

What recommendations would you have for solving this issue or how to fix it?

**Reply by Fancy2209** [reply]
As far as I can tell fhe file wasn't preserved, if isn't there, it's gone

**Comment by valepu** [comment]
Disclaimer: I don't know your web expertise but I'm gona use some technical terms here

I had some free time and I've checked the flash player in the page and decompiled it with http://www.nowrap.de/flare.html 
The actionscript ultimately points to http://www.vidiac.com/handlers/buildflash.ashx?fileid=... (one of the videos would be fileid=afde1a37-2647-4a5b-b3c6-3bf37decbffd) but no url like this has been saved under the internet archive https://web.archive.org/web/*/vidiac.com/handlers/buildflash*

I noticed there's a Windows Media Player version too (link right above the video) in which the wma plugin points to urls like this http://www.vidiac.com/playlistbuilder/buildasx.ashx?fileid=afde1a37-2647-4a5b-b3c6-3bf37decbffd&t=82ee5a08-0791-4f41-8144-99ed004d0cce these URLs exist under the internet archive: https://web.archive.org/web/*/vidiac.com/playlistbuilder* but they return empty pages (probably they expect to be contacted by an old version of windows media player and not by a browser, or they haven't been saved correctly by the Internet Archive). 

This website had a slightly complicated way of handling videos (but it's understandable since back then it wasn't so straightforward to play videos directly on web pages) and this caused the internet archive to not be able to save the files. 
I'm pretty confident there's no way to recover the flash version, maybe you have a very tiny chance with the wmp version but it's highly unlikely. You should get a Virtual Machine with an old version of Windows, Media Player and IE and try those URLs I mentioned above. But it will probably be a waste of time

Good luck :)
